conda.recipe/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: dask
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '')}}
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
+
+source:
+  git_url: ../
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolz
+    - psutil
+    - chest
+    - heapdict
```

Comment:
Is pyzmq a "build" dependency too? I have it under the `run` field.

I think that the build dependencies are removed after the package is built. Then the runtime dependencies are installed.